app/uk/gov/hmrc/exports/mongock/changesets/CacheChangelog.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmrc.exports.mongock.changesets;
+
+import com.github.cloudyrock.mongock.ChangeLog;
+import com.github.cloudyrock.mongock.ChangeSet;
+import com.mongodb.client.MongoDatabase;
+
+@ChangeLog
+public class CacheChangelog {
+    private String collection = "declarations";
+
+    @ChangeSet(order = "001", id = "Deckaration DB Baseline", author = "Patryk Rudnicki")
```

Comment:
you need to keep the same changeset info, otherwise it will apply it again:

```
@ChangeSet(order = "001", id = "Exports DB Baseline", author = "Paulo Monteiro")
```

Reply:
But double check locally

Reply:
I change it. Thanks for the review. I'll double check it locally.